### PR TITLE
add the task ID when print

### PR DESCRIPTION
**Pull request by xyw0537**
change the name of Cloudlet ID to Job ID and add the task ID so that
the process of clustering can be seen more clearly.

### CODE REVIEW

File: examples/org/workflowsim/examples/WorkflowSimBasicExample1.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
 import java.util.Calendar;
 import java.util.LinkedList;
 import java.util.List;
+
+import org.apache.commons.math3.util.MultidimensionalCounter.Iterator;
```
Comment (chenww05) [inline examples/org/workflowsim/examples/WorkflowSimBasicExample1.java]: No need of Iterator.

File: examples/org/workflowsim/examples/WorkflowSimBasicExample1.java [hunk 2]
```diff
@@ -252,13 +256,20 @@ protected static void printJobList(List<Job> list) {
...
                 + "Data center ID" + indent + "VM ID" + indent + indent
                 + "Time" + indent + "Start Time" + indent + "Finish Time" + indent + "Depth");
         DecimalFormat dft = new DecimalFormat("###.##");
         for (int i = 0; i < size; i++) {
             job = list.get(i);
-            Log.print(indent + job.getCloudletId() + indent + indent);
+            Log.print(indent + job.getCloudletId() + indent +indent);
+            if(job.getCloudletId()==size-1)
+            	Log.print("Stage-in");
+            for(java.util.Iterator<Task> it=job.getTaskList().iterator();it.hasNext();){
```
Comment (chenww05) [inline examples/org/workflowsim/examples/WorkflowSimBasicExample1.java]: Use for (Task task : job.getTaskList())

File: examples/org/workflowsim/examples/WorkflowSimBasicExample1.java [hunk 3]
```diff
@@ -252,13 +256,20 @@ protected static void printJobList(List<Job> list) {
...
-        Log.printLine("Cloudlet ID" + indent + "STATUS" + indent
+        Log.printLine("Job ID" + indent + "Task ID" + indent +"STATUS" + indent
                 + "Data center ID" + indent + "VM ID" + indent + indent
                 + "Time" + indent + "Start Time" + indent + "Finish Time" + indent + "Depth");
         DecimalFormat dft = new DecimalFormat("###.##");
         for (int i = 0; i < size; i++) {
             job = list.get(i);
-            Log.print(indent + job.getCloudletId() + indent + indent);
+            Log.print(indent + job.getCloudletId() + indent +indent);
+            if(job.getCloudletId()==size-1)
```
Comment (chenww05) [inline examples/org/workflowsim/examples/WorkflowSimBasicExample1.java]: I would prefer : if (job.getClassType() == ClassType.STAGE_IN.value) {

**Comment by chenww05** [comment]
Hi xyw0537, 

Could you update your commits based on my notes? (To make it consistent with other codes) Thanks.

**Comment by xyw0537** [comment]
ok, i am trying to build a pegasus system in our lab and  i have met some questions. could you tell me your email?

**Comment by chenww05** [comment]
sure wchenpublic@gmail.com but for pegasus specific info please send it 
to pegasus support team : pegasus-support@isi.edu 
<mailto:pegasus-support@isi.edu>

**Pull request closed without merging**